core/network/types/collator_messages.hpp

```diff
@@ -101,8 +97,8 @@ namespace kagome::network {
   struct CandidateReceipt {
     SCALE_TIE(2);
 
-    CandidateDescriptor descriptor;          /// Candidate descriptor
-    primitives::BlockHash commitments_hash;  /// Hash of candidate commitments
+    CandidateDescriptor descriptor;        /// Candidate descriptor
+    primitives::BlockHash candidate_hash;  /// Hash of candidate commitments
```

Comment:
It's not `pov_hash` nor `para_head`.
```suggestion
    primitives::BlockHash commitments_hash;  /// Hash of candidate commitments
```